examples/event-sources/gcp-pubsub-workflow-identity.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
instead of creating a separate file for workflow identity example, can you put it under existing example https://github.com/argoproj/argo-events/blob/25db621a6c150465442fff27a7a418c4f4402d08/examples/event-sources/gcp-pubsub.yaml

Something like,
```

apiVersion: argoproj.io/v1alpha1
kind: EventSource
metadata:
  name: gcp-pubsub-event-source
spec:
  type: pubsub
  pubSub:
    example:
      # jsonBody specifies that all event body payload coming from this
      # source will be JSON
      jsonBody: true
      # id of your project
      projectID: argo-events-XXXXX
      # (optional) id of project for topic, same as projectID by default
      # topicProjectID: "project-id"
      # topic name
      topic: test
      # Refers to path of the credential file that is mounted in the gateway pod.
      credentialsFile: /creds/key.json

#    example-workload-identity:
#      # jsonBody specifies that all event body payload coming from this
#      # source will be JSON
#      jsonBody: true
#      # id of your project
#      projectID: argo-events-XXXXX
#      # (optional) id of project for topic, same as projectID by default
#      # topicProjectID: "project-id"
#      # topic name
#      topic: test
#      # Empty credentials file when using Workflow Identity
#      credentialsFile: ""
#      # If enableWorkflowIdentity is true, the projects uses Workflow Identity for authentication
#      enableWorkflowIdentity: true
#


```

Reply:
done